config/kraft/README.md

```diff
@@ -114,9 +114,7 @@ We don't support any kind of upgrade right now, either to or from KRaft mode.  T
 
 Finally, the following Kafka features have not yet been fully implemented:
 
-* Support for certain security features: configuring a KRaft-based Authorizer, setting up SCRAM, delegation tokens, and so forth
-  (although note that you can use authorizers such as `kafka.security.authorizer.AclAuthorizer` with KRaft clusters, even
-  if they are ZooKeeper-based: simply define `authorizer.class.name` and configure the authorizer as you normally would).
+* Support for certain security features: setting up SCRAM, delegation tokens, and so forth.
```

Comment:
Do you have the full list? :) If not, let's get this merged and then follow up with future improvements. Since 3.2.0 is imminent, I wanted to fix this.

Reply:
This is what is missing according to KIP-833:
```
Configuring SCRAM users via the administrative API
Supporting JBOD configurations with multiple storage directories
Modifying certain dynamic configurations on the standalone KRaft controller
Delegation tokens
Upgrade from ZooKeeper mode
```